lib/generate/collector/product.js

```diff
@@ -1,20 +1,20 @@
-const merge = require('lodash.merge');
+const merge = require(`lodash.merge`);
```

Comment:
What's the reason for this change from quotes to ticks?

Reply:
Hello Fredden. Thank you for your question. This was changed during my debugging. I had added in several lines to print to console the status and progress of processes at different points in the console, including API requests and responses. During this time, my print statements were displaying the variable name (e.g. ${some_variable}) I used, rather than the value of the variable as I had intended. both single quotes and double quotes caused the console print to display th variable name instead of the variable value, but using backticks enabled the variable value to be displayed.

Reply:
Thanks for confirming that this is not an intended change. Please can you adjust this so that the changes in this pull request only show what you intended to change.